creating-pull-requests.md

```diff
@@ -0,0 +1,131 @@
+
+Note: the terms _Pull Request_ and _Merge Request_ can be used interchangeably. The same is true for the terms _issue_, _ticket_, and _stories_.
+
+## Before you start working
+
+Make sure that there is an issue created and assigned to you. Work on a feature branch that is named accordingly. It might also be a good idea to validate the approach you intend to take in case it is a big or complex issue.
+
+## When to create a Pull Request?
+
+The two main reasons to create a pull request are when your work is complete, or when you want early feedback on your work.
+
+### When your work is complete
+
+You finished the work on an issue in a dedicated feature branch, and you’re ready to have your code merged into develop. Before doing that, others will review it according to the Frontend Pull Request Reviewer Checklist. This process is also useful for others to learn about the changes you have made. The pull request can be merged after the work is reviewed and all of the feedback has been addressed.
+
+### When you want early feedback on your work
+
+If you have been assigned a task, but are not 100% sure about the approach you are taking and want some feedback before you continue down that road, please make sure that:
+
+- You have added WIP to the title to make it clear that it is still a work in progress
+- It is clear which specific code or features you’d like to have feedback on
+- You indicated which people you want to review your code
+
+After the feedback, you can either decline the PR and start over or continue with the work in the same PR. You can keep committing in the same branch with updates, so the reviewers can keep track of your work. Just make sure you let them know when it’s ready to review again by changing the PR description and mentioning them. Remove WIP from the title when it’s ready and make sure the reviewers are notified.
+
+## How to create a PR?
+
+There are a couple of things to pay attention to when creating a PR, such as having a correct title, and all the necessary information in the description.
+
+### Have a correct title
+
+Here are some guidelines for a correct title:
+
+- Should contain issue identification, such as Jira issue key or Assembla ticket number (when related to a single issue).
+- Should be descriptive enough. A good rule of thumb is to make it the same as the issue summary.
+- No emoticons. They add noise and don’t render properly in the PR list on some platforms.
+
+### Make sure the description is clear
+
+#### Link all issues
+
+When your PR covers multiple issues (e.g. combined QA fixes), those links should be present in the description from the commit log.
+
+#### Provide context
+
+If it’s a complex architecture PR, explain the main contexts in the PR description or link out to relevant technical documents.
+
+#### Add design links or screenshots
+
+If it’s a UI PR, link to the designs or add a screenshot so reviewers can clearly understand the scope of the PR, and compare it with that.
```

Comment:
Shouldn't these be handled/available in the ticket?

Reply:
I agree, the ticket should be the only source of truth for a task. The PR description should only include matters, which help understand the specific changes/additions in the PR.
However, to prevent being a burden, we have to make sure that whatever PR tool we're using can link to the ticket in a single click.